addon/components/labs-bbl-lookup.js

```diff
@@ -35,28 +35,37 @@ export default Component.extend({
 
   actions: {
     validate() {
-      const boro = this.get('boro');
-      const block = this.get('block');
-      const lot = this.get('lot');
+      const boro = this.boro;
+      const block = this.block;
+      const lot = this.lot;
 
-      const validBoro = (boro !== '');
-      const validBlock = ((block !== '') && (parseInt(block, 10) < 100000) && (parseInt(block, 10) > 0));
-      const validLot = ((lot !== '') && (parseInt(lot, 10) < 10000) && (parseInt(lot, 10) > 0));
+      const validBoro = boro !== '';
+      const validBlock =
+        block !== '' && parseInt(block, 10) < 100000 && parseInt(block, 10) > 0;
+      const validLot =
+        lot !== '' && parseInt(lot, 10) < 10000 && parseInt(lot, 10) > 0;
 
       this.set('validBlock', validBoro && validBlock);
       this.set('validLot', validBoro && validBlock && validLot);
 
-      const submitText = (validBlock && !validLot) ? 'Go to Block' : 'Go to Lot';
+      const submitText = validBlock && !validLot ? 'Go to Block' : 'Go to Lot';
       this.set('submitText', submitText);
     },
 
     handleSubmit() {
-      const { boro: { code }, block, lot } = this.getProperties('boro', 'block', 'lot');
-      const validBlock = this.get('validBlock');
-      const validLot = this.get('validLot');
+      const {
+        boro: { code },
+        block,
+        lot,
+      } = this;
+      const validBlock = this.validBlock;
+      const validLot = this.validLot;
 
       if (validBlock && !validLot) {
-        const SQL = `SELECT the_geom FROM dof_dtm_block_centroids WHERE block= ${parseInt(block, 10)} AND borocode = '${code}'`;
+        const SQL = `SELECT the_geom FROM dof_dtm_block_centroids WHERE block= ${parseInt(
+          block,
+          10
+        )} AND borocode = '${code}'`;
```

Comment:
```suggestion
        )} AND boro = '${code}'`;
```

This column changed recently...